ProjectSetup/Controllers/V1/PostController.cs

```diff
@@ -48,7 +52,11 @@ public async Task<ActionResult<Post>> Get([FromRoute] Guid postId)
 		[CustomExceptionFilter(typeof(PostBadRequestException), HttpStatusCode.BadRequest)]
 		public async Task<IActionResult> Create([FromBody] CreatePostRequest postRequest)
 		{
-			var post = await _repository.Post.CreatePost(postRequest);
+			Post mappedPost = _mapper.Map<Post>(postRequest);
+
+			mappedPost.CreatedBy = Guid.Parse(HttpContext.GetUserId());
```

Comment:
You can edit GetUserId method to return Guid and parse this parameter inside, or create new method which will call GetUserId and return you Guid parameter.

Reply:
I've fixed it.